data/strings/types_strings.txt

```diff
@@ -10910,6 +10910,85 @@
     zh-Hans = 退出
     zh-Hant = 出口
 
+  [type.fee.yes]
+    en = Paid
```

Comment:
so for an `amenity=parking fee=yes` feature it'll be `Paid Parking • Paid` in the PP subtitle, which doesn't look very good...

@vng does search use type's translation for searching? i.e. does presence of "Paid" in the translation make a difference?
if not, then we can probably drop it and leave just "Parking"

Reply:
AFAIK there are no parking-fee types in categories so currently you cannot search for "Paid Parking" exactly. However when you search for "Parking" you will see the description as "Paid Parking" if that's the case.

Reply:
I think we should add the "Paid"/"Free" indicator to search results too and simplify the "Paid Parking" strings to "Parking" so it would read as "Parking • Paid" or "Parking • Free" both in the PlacePage and in the search results